package.json

```diff
@@ -11,6 +11,7 @@
     "start": "node index.js",
     "babel-dev": "babel public/js/*.js --presets es2015 --out-file public/js/main-compiled.js --source-maps --ignore *-compiled.js --watch",
     "dev": "nodemon index.js",
+    "tailwind-gen": "./node_modules/.bin/tailwind.cmd build ./tailwind-gen.css -c ./tailwind.js -o ./public/css/tailwind.css",
```

Comment:
Guessing you're on windows, `tailwind.cmd` doesn't exist on *nix systems it seems, so wondering if we can update this to simply use `./node_modules/.bin/tailwind`